cypress/support/pages/yves/content/content-page.ts

```diff
@@ -0,0 +1,22 @@
+import { REPOSITORIES, autoWired } from '@utils';
+import { inject, injectable } from 'inversify';
+
+import { YvesPage } from '@pages/yves';
+import { ContentRepository } from './content-repository';
+
+@injectable()
+@autoWired
+export class ContentPage extends YvesPage {
+  @inject(REPOSITORIES.ContentRepository) private repository: ContentRepository;
+
+  protected PAGE_URL = '/search';
+
+  searchCmsPageFromSuggestions = (params: SearchParams): void => {
```

Comment:
maybe `findAndSelectSuggestedCmsPage` instead of `searchCmsPageFromSuggestions` ?